wsidicom/conceptcode.py

```diff
@@ -13,14 +13,17 @@
 #    limitations under the License.
 
 from dataclasses import dataclass
-from typing import List, Dict, Optional
+from typing import List, Dict, Optional, Type, TypeVar
 
 from pydicom.dataset import Dataset
 from pydicom.sequence import Sequence as DicomSequence
 from pydicom.sr.codedict import codes
 from pydicom.sr.coding import Code
 
 
+CONCEPTCODE = TypeVar('CONCEPTCODE', bound='ConceptCode')
```

Comment:
It is recommended to use CapWords for Typevars: https://peps.python.org/pep-0008/#type-variable-names since that clashes with class name, you need something else. Even though it is possible to separate names by altering caps in Python, it should be avoided. So I would suggest something like: `ConceptCodeType` instead.